tests/tests/relayer.rs

```diff
@@ -309,6 +309,10 @@ async fn handle(
             let r = mock.get_block_number().await.unwrap();
             json!({ "id": id, "jsonrpc": "2.0", "result": r })
         }
+        "eth_getBlockByNumber" => {
```

Comment:
I guess we can remove `eth_blockNumber` now

Reply:
Yep, that's true. If we need it again in the future for whatever reason, we can add it back.